src/main/java/org/prebid/server/bidder/brightroll/BrightrollBidder.java

```diff
@@ -121,6 +126,10 @@ private String getAndValidateImpExt(Imp imp) {
             throw new PreBidException("publisher is empty");
         }
 
+        if (!supportedVendors.contains(publisher)) {
```

Comment:
I think, you can use `!publisherIdToOverride.keys().contains(publisher)` and remove supportedVendors collections from constructor. 
Or just create `supportedVendors` from `publisherIdToOverride.keys()`.

Reply:
Thanks for reviewing @DGarbar , changed to use vendors from map directly.